test/parallel/test-cli-node-options.js

```diff
@@ -24,6 +24,13 @@ expect('--throw-deprecation', 'B\n');
 expect('--zero-fill-buffers', 'B\n');
 expect('--v8-pool-size=10', 'B\n');
 expect('--trace-event-categories node', 'B\n');
+expect('--perf-basic-prof', 'B\n');
+
+const archMatched = ['arm', 'x64', 'mips'].includes(process.arch);
+if (common.isLinux && !common.isWindows && archMatched) {
```

Comment:
Aren't these mutually exclusive? If it *is* linux, then surely it is *not* windows? I mention, because I  think the `['arm', 'x64', 'mips'].includes(process.arch)` could be substituted for the `!common.isWindows && archMatched` and it would all fit on one line, and not imply that a system can be simultaneously Windows and Linux. Otherwise, LGTM

Reply:
Thank you for review. I removed redundant condition: `!common.isWindows`.